lib/openstax/rescue_from/configuration.rb

```diff
@@ -2,10 +2,54 @@ module OpenStax
   module RescueFrom
     class Configuration
 
-      # attr_accessor, attr_reader, attr_writers for variables
+      attr_accessor :raise_exceptions, :system_logger, :notifier,
+        :html_template_path, :layout_name, :application_name, :exception_status_codes,
+        :non_notifying_exceptions, :exception_extras,:friendly_status_messages
 
       def initialize
-        # set default values for variables
+        @raise_exceptions = false
+        @application_name = 'Tutor'
+        @system_logger = Rails.logger
```

Comment:
Sure but it may not be the "system" logger that is configured -- I'm cool with RescueFrom having its own Logger that calls off to some other configured logger -- don't want to change that.  Just think we don't need to name it "system_logger".